plugins/zns/zns.c

```diff
@@ -154,14 +156,19 @@ static int zns_mgmt_send(int argc, char **argv, struct command *cmd, struct plug
 		__u64	zslba;
 		__u32	namespace_id;
 		bool	select_all;
+		__u32	timeout;
 	};
 
-	struct config cfg = {};
+	struct config cfg = {
+		/* Longer (90 second) default timeout for zone_reset */
```

Comment:
Hi,

It may be better to use the default timeout value. Otherwise it may confuse people using the tool that some commands allows a longer timeout, while others use the default timeout value (30 secs?)

Reply:
Howdy @MatiasBjorling,

The default value here is set similarly to how 'nvme format' handles the timeout parameter (90 seconds here vs. 60 seconds for format). Zone-reset seems to be the only zone management command that will regularly require a longer timeout than the default value, particularly with the -a option and a large namespace. 

Because of this, I've set the default to a higher value in that case to save the user some headache. With or without the extended default, users would have to find out about any timeout issue and set it manually with the new parameter.

Reply:
If you are finding that the system's default time is insufficient, it's fine with me to let the tooling pick one that is more appropriate. That said, even the default 30 seconds seems like an awfully long time for a zone reset operation. Is that normal?